witwidget/notebook/jupyter/js/lib/labplugin.js

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+var witwidget = require('./index');
+var base = require('@jupyter-widgets/base');
+
+module.exports = {
+  id: 'wit-widget',
+  requires: [base.IJupyterWidgetRegistry],
+  activate: function(app, widgets) {
+    widgets.registerWidget({
+      name: 'wit-widget',
+      version: '0.1.0',
```

Comment:
should this be updated to actual witwidget version? it has been working like this so far.

Reply:
going to keep it as-is and working in this PR that moves the code over. we can revisit in its own work